libfuse/tlf.go

```diff
@@ -163,7 +168,9 @@ func (tlf *TLF) Lookup(ctx context.Context, req *fuse.LookupRequest, resp *fuse.
 }
 
 // Create implements the fs.NodeCreater interface for TLF.
-func (tlf *TLF) Create(ctx context.Context, req *fuse.CreateRequest, resp *fuse.CreateResponse) (fs.Node, fs.Handle, error) {
+func (tlf *TLF) Create(ctx context.Context, req *fuse.CreateRequest, resp *fuse.CreateResponse) (_ fs.Node, _ fs.Handle, err error) {
+	tlf.log().CDebugf(ctx, "TLF Create %s", req.Name)
+	defer func() { tlf.folder.reportErr(ctx, libkbfs.WriteMode, err) }()
```

Comment:
If the code reaches `dir.Create`, two errors will be reported for the same operation.  Maybe you should only report this one when the `loadDir` fails?